app/main.py

```diff
@@ -0,0 +1,66 @@
+import json
+from confluent_kafka import Consumer
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+server = os.getenv("KAFKA_SERVER")
+username = os.getenv("KAFKA_KEY")
+password = os.getenv("KAFKA_SECRET")
+
```

Comment:
looks great, like how you have used the .env file to store the sensitive information

Reply:
Thanks